lms/djangoapps/django_comment_client/tests/test_utils.py

```diff
@@ -374,7 +380,6 @@ def setUp(self):
         )
         # Courses get a default discussion topic on creation, so remove it
         self.course.discussion_topics = {}
-        self.course.save()
```

Comment:
Note to reviewers: these save calls aren't doing anything-- in order to actually save in the modulestore, you have to call update_item.